Uppsala/UppEt29.xml

```diff
@@ -0,0 +1,280 @@
+                                ደብተራ፡ ለኤዶም፡ 
+                                <gap reason="lost" unit="lines" quantity="1"/>
+                                ለገብርኪ፡ እምኮከበ፡
+                                <gap reason="lost" unit="lines" quantity="1"/>
+                                <sic>እመ፡</sic> ቀሲስ፡ አንቲ፡ ሀብኒ፡ ን<gap reason="lost" unit="chars" quantity="2"/>
+                            </incipit>
+                        </msItem>
+                            
+                        <msItem xml:id="ms_i2">
+                            <locus from="14r" to="35v"/>
+                            <title type="complete" ref="LIT2885RepCh166"/>
+                            <incipit xml:lang="gez">
+                                <gap reason="lost" unit="lines" quantity="2"/>ዱስ፡ ወጠነ፡ ማኅሌተ፡ ክቡር፡ 
+                                <gap reason="lost" unit="chars" quantity="4"/> 
+                                ወዘ<gap reason="lost" unit="chars" quantity="2"/><hi rend="rubric">ልዑ፡ 
+                                    ድር<gap reason="lost" unit="chars" quantity="5"/></hi>ዕ፡ ልሳንየ፡ 
+                            </incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i3">
+                            <locus from="37v" to="38r"/>
+                            <title type="complete" ref="LIT7121PrayerChildbirth"/>
+                            <incipit xml:lang="gez">
+                                በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ጸሎት፡ በእንተ፡ ሕማመ፡ ወሊድ፡ 
+                                ወሶበ፡ ታስተበርክ፡ ብእሲት፡ ለወሊድ፡ ወትብል፡ ሀበኒ፡ ሣህለከ፡ ወምሕረተከ፡ 
+                            </incipit>
+                        </msItem>
+                        
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">38</measure>
+                                    <measure unit="page" type="blank">2</measure>
+                                    <measure unit="quire">5</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>85</height>
+                                        <width>55</width>
+                                    </dimensions>
+                                </extent>
+                                
+                                <foliation>Foliation in pencil in the upper outer corner.</foliation>
+                                
+                                <collation>
+                                    
+                                    <list>
+                                        <item xml:id="q1" n="1">
```

Comment:
if this is a quire consisting of guardleaves, I have marked (and found) this quire as `<item xml:id="q1" n="A">` (and similarly, the last quire is, e.g., `<item xml:id="q12" n="B">`). I do not know how widespread this usage is, perhaps it comes from the way guardleaf quires were counted in Ethio-SPaRe. I have not found any specific instruction on this in the guidelines. Are there preferences between `n="A"` and `n="1"`?

Reply:
I prefer n="A", if it is clearly a guard leaf.

Reply:
My thought was that it's better to describe without adding too much interpretation, but I'm open to renumbering the flyleaf quires :)